src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -201,12 +201,45 @@ private static OpenApiResponses GetOpenApiResponses(MethodInfo method, EndpointM
                 responseContent[contentType] = new OpenApiMediaType();
             }
 
-            responses[statusCode] = new OpenApiResponse { Content = responseContent };
+            responses[statusCode] = new OpenApiResponse
+            {
+                Content = responseContent,
+                Description = GetDescription(statusCode)
+            };
         }
 
         return responses;
     }
 
+    private static string GetDescription(string statusCode)
+    {
+        string first = statusCode[..1];
+        if (first == "1")
+        {
+            return "information";
```

Comment:
Try rewriting this as a [switch expression](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression).